app/config/config_test.php

```diff
@@ -30,7 +30,8 @@
 $container->loadFromExtension('framework', [
     'test'    => true,
     'session' => [
-        'storage_id' => 'session.storage.filesystem',
+        'storage_id' => 'session.storage.mock_file',
```

Comment:
@escopecz this change is based on the [docs here](https://github.com/liip/LiipFunctionalTestBundle/blob/master/doc/installation.md)